src/components/Modal.js

```diff
@@ -182,9 +187,16 @@ class Modal extends React.Component {
   render () {
     const theme = StyleUtils.mergeTheme(this.props.theme, this.props.color);
     const styles = this.styles(theme);
+    const mergedFocusTrapProps = {
+      focusTrapOptions: {
+        clickOutsideDeactivates: true
+      },
+      paused: false,
```

Comment:
Why did you include this line?

Reply:
It looks like MXFocusTrap will override this prop with it's state: https://github.com/mxenabled/mx-react-components/blob/66a588289d0bcd656b81975b9866a4398b399615/src/components/MXFocusTrap.js#L34

Reply:
I was thinking that it was required because of the the `MXFocusTrap` stuff we did but it looks like `MxFocusTrap` defaults to false on its own.  I'll remove it.